File: _IFSP/LP1/Reposicao_2aul/EX1-Empresa-r/EX1.php
<?php
echo "Funcionário: ";
$nome = trim(fgets(STDIN));
echo "\n";

echo "Salário: ";
$salario = trim(fgets(STDIN));
echo "\n";
$sa = (float)$salario;
$pct;

if ($sa <= 400.00) {
    $pct = 15;
} elseif ($sa <= 700.00) {
    $pct = 12;
} elseif ($sa <= 1000.00) {
    $pct = 10;
} elseif ($sa <= 1800.00) {
    $pct = 7;
} elseif ($sa <= 2500.00) {
    $pct = 4;
} else {
    $pct = 0;
}

$ns = $sa + ($sa * $pct / 100);

echo "Nome do funcionário: $nome\n";
echo "% de aumento: $pct%\n";
echo "Salário atual: R$ " . number_format($sa, 2) . "\n";
echo "Novo salário: R$ " . number_format($ns, 2) . "\n";
?>

File: 01_IFSP/LP1/Reposicao_2aul/EX1-Empresa-r/EX1.cpp
#include <iostream>
#include <iomanip>
#include <string>

int main() {
    std::string nome;
    float sa, ns;
    int pct;

    std::cout << "Funcionário: ";
    std::getline(std::cin, nome);
    std::cout << ""<< std::endl;

    std::cout << "Salário: ";
    std::cin >> sa;

    if (sa <= 400.00f) {
        pct = 15;
    } else if (sa <= 700.00f) {
        pct = 12;
    } else if (sa <= 1000.00f) {
        pct = 10;
    } else if (sa <= 1800.00f) {
        pct = 7;
    } else if (sa <= 2500.00f) {
        pct = 4;
    } else {
        pct = 0;
    }

    ns = sa + (sa * pct / 100);
    std::cout << ""<< std::endl;
    std::cout << "Nome do funcionário: " << nome << std::endl;
    std::cout << "% de aumento: " << pct << "%" << std::endl;
    std::cout << "Salário atual: R$ " << std::fixed << std::setprecision(2) << sa << std::endl;
    std::cout << "Novo salário: R$ " << std::fixed << std::setprecision(2) << ns << std::endl;

    return 0;
}
